client/signup/jetpack-connect/site-url-input.jsx

```diff
@@ -22,8 +22,9 @@ export default React.createClass( {
 	},
 
 	onChange( event ) {
+		const url = event.target.value;
 		this.setState( {
-			value: event.target.value
+			value: url.replace(/\/$/, "")
```

Comment:
ok... I'll change it, but... what's the point of having a module that replace, literally, a 5 char regexp & a `.replace`? Didn't we learned anything with the left padding debacle? :D